composer.json

```diff
@@ -1,66 +1,76 @@
 {
-    "name": "doctrine/migrations",
-    "type": "library",
-    "description": "PHP Doctrine Migrations project offer additional functionality on top of the database abstraction layer (DBAL) for versioning your database schema and easily deploying changes to it. It is a very easy to use and a powerful tool.",
-    "keywords": ["php", "database", "migrations", "dbal"],
-    "homepage": "https://www.doctrine-project.org/projects/migrations.html",
-    "license": "MIT",
-    "authors": [
-        {"name": "Benjamin Eberlei", "email": "kontakt@beberlei.de"},
-        {"name": "Jonathan Wage", "email": "jonwage@gmail.com"},
-        {"name": "Michael Simonson", "email": "contact@mikesimonson.com" }
```

Comment:
What's wrong with having multiple properties per line here? it still fits under a reasonable name. Besides, others might not have that `npm` package you used. Maybe we should adopt https://github.com/ergebnis/composer-normalize and put it in the CI? That would take care of that kind of argument over indent size, etc.